Sources/MapboxNavigation/Feedback/ActiveNavigationFeedbackType+FeedbackItem.swift

```diff
@@ -6,30 +6,24 @@ public extension ActiveNavigationFeedbackType {
     // TODO: Localize these strings
     var title: String {
         switch self {
-        case .general:
-            return "Feedback"
```

Comment:
I always assumed this “general” category was used for end-of-route feedback, but it turns out that the rating and comment go into a cancel event instead of a feedback event. Who knew? 🤷‍♂️